demo/lib/distro.bash

```diff
@@ -529,6 +529,10 @@ fedora-install-containerd-pre() {
     fedora-install-containernetworking-plugins
 }
 
+fedora-install-containerd-post() {
+    vm-command "systemctl enable containerd"
```

Comment:
not related to kernel cmdline 🧐 A separate update the commit message?

Reply:
Yeah, I wonder about same. IMO, there is no need for two PRs, but it would make sense to have those as two separate commits, both with proprer commit messages, if for the related hunk the current message is used. And maybe the PR title could be something a bit more appropriate, like "e2e: fix for isolcpus tests on fedora"...